pymesync.py

```diff
@@ -0,0 +1,70 @@
+"""
+pymesync - Python TimeSync Module
+
+Allows for interactions with the TimeSync API
+- timesync.send_time(baseurl) -- Sends time to baseurl (TimeSync)
+"""
+import json
+import requests
+
+
+class TimeSync(object):
+
+    def __init__(self, baseurl, user, password, auth_type):
+        self.baseurl = baseurl
+        self.user = user
+        self.password = password
+        self.auth_type = auth_type
+
+    def send_time(self, parameter_dict):
+        """
+        send_time(parameter_dict)
+
+        Sends a POST request in a JSON body to TimeSync, returns that body if
+        successful or an error if not.
+        parameter_dict - python dict containing time info.
+        """
+
+        values = {
+            'auth': self._auth(),
+            'object': {
+                'duration': parameter_dict['duration'],
+                'user': parameter_dict['user'],
+                'project': parameter_dict['project'],
+                'activities': parameter_dict['activities'],
+                'date_worked': parameter_dict['date_worked'],
+                'notes': parameter_dict['notes'],
+                'issue_uri': parameter_dict['issue_uri'],
```

Comment:
Just as a heads up, you may want to either use ```parameter_dict.get(key)``` or check for the existence of the keys before you send it in otherwise you'll get KeyError exceptions. The plus side to this approach is you can use default values as the secondary arguments to the ```.get``` method if the key doesn't exist.

However, if everything you're looking for is in parameter dict, this may be a bit cleaner:
```
values = {
            'auth': self._auth(),
            'object': parameter_dict,
}
```

Anyways, food for thought.